cli/src/pcluster/config/cluster_config.py

```diff
@@ -883,10 +894,9 @@ def __init__(self, local_storage: LocalStorage = None, **kwargs):
 class BaseQueue(Resource):
     """Represent the generic Queue resource."""
 
-    def __init__(self, name: str, networking: QueueNetworking, capacity_type: str = None):
+    def __init__(self, name: str, capacity_type: str = None):
```

Comment:
def get_common_user_data_env(node: Union[HeadNode, BaseQueue], config: BaseClusterConfig) -> dict:
in cdk_builder_utils.py also need to be changed accordingly.

Reply:
good catch, updated